test/config/destination.json

```diff
@@ -1,7 +1,7 @@
 {
     "catalog":"test/catalog/destination.json",
     "protocol":"http",
-    "backend":"/tmp/transfer2go/destination",
+    "backend":"test/destination",
```

Comment:
Why do you keep changing this to test/destination. I don't want test files to be in a code repository. That's why I put /tmp area for that.